ui/k-info-panel/show_circuit.kytos

```diff
@@ -733,6 +735,33 @@ module.exports = {
 
       return result;
     },
+    redeployEvc: function() {
+      var _this = this;
+
+      let id = this.basics["ID"]["value"];
+      let name = this.basics["Name"]["value"];
+      var request = $.ajax({
+        url: this.$kytos_server_api + "kytos/mef_eline/v2/evc/" + id + "/redeploy",
+        type:"PATCH",
+        data: JSON.stringify({"empty": true}),
```

Comment:
@hwelsters, this `{"empty": true}` body isn't [actually required to be sent](https://kytos-ng.github.io/api/mef_eline.html#operation/redeploy), could you try to remove it?